marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mastering/default/lib.sjs

```diff
@@ -21,6 +21,7 @@ const masteringConsts = require("/com.marklogic.smart-mastering/constants.xqy");
 const requiredOptionProperties = ['matchOptions', 'mergeOptions'];
 const ps = require('/MarkLogic/provenance');
 const op = require('/MarkLogic/optic');
+const emptySequence = Sequence.from([]);
```

Comment:
I think this and the other two application code changes really need to be in a separate PR, ideally associated with a separate JIRA ticket that captures the problems these are fixing. Also, those can in theory be moved along faster than the rest of this PR - i.e. the rest of this PR doesn't seem as important as these 3 changes here, since the rest of the PR is just updates to example projects.